src/libraries/System.Net.Http/src/System/Net/Http/HttpTelemetry.cs

```diff
@@ -50,18 +50,34 @@ public void RequestStart(HttpRequestMessage request)
                 request.VersionPolicy);
         }
 
-        [Event(2, Level = EventLevel.Informational)]
-        public void RequestStop()
+        [NonEvent]
+        public void RequestStop(HttpResponseMessage? response)
+        {
+            RequestStop(response is null ? -1 : (int)response.StatusCode);
+        }
+
+        [Event(2, Level = EventLevel.Informational, Version = 1)]
```

Comment:
I'm not familiar with how we think about these events and versioning.  Is it a breaking change to version the event and add data?  Is that something we normally do in servicing?  cc @noahfalk @tarekgh

Reply:
https://learn.microsoft.com/en-us/dotnet/core/diagnostics/eventsource-instrumentation#best-practices

```
Attempt to keep EventSource object back compatible and version them appropriately. The default version for an event is 0. The version can be changed by setting [EventAttribute.Version](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.tracing.eventattribute.version). Change the version of an event whenever you change the data that is serialized with it. Always add new serialized data to the end of the event declaration, that is, at the end of the list of method parameters. If this isn't possible, create a new event with a new ID to replace the old one.
```

CC @davmason

Reply:
It is not considered a breaking change to add fields at the end of an event and @brianrob confirmed earlier that we have done this type of telemetry addition in servicing before without any issue.

Reply:
Correct - this is not a breaking change.

Reply:
Thanks folks!